dev/local-environment/setup.sh

```diff
@@ -434,6 +434,7 @@ create_docker_compose() {
         cat ./modules/postgres.txt >> docker-compose.yml
         cat ./modules/partner-chains-external-node.txt >> docker-compose.yml
         cat ./modules/pc-contracts-cli.txt >> docker-compose.yml
```

Comment:
Shouldn't we delete pc-contracts-cli module?

Reply:
It would ruin E2E-tests. It is your job to modify them when this is merged. Then we can clean up. Good luck, sir!